resources/vbmc/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM registry.hub.docker.com/library/python:3.7
 
 RUN apt update && \
-    apt install -y libvirt-dev && \
-    pip3 install virtualbmc
+    apt install -y curl libvirt-dev && \
+    curl -Lf -o /tmp/uc.txt https://releases.openstack.org/constraints/upper/master && \
+    pip3 install virtualbmc -c /tmp/uc.txt
```

Comment:
This seems reasonable, but I wonder if we might also consider switching to the RDO packages, since we already use those for the ironic images:

https://github.com/metal3-io/ironic-image/blob/master/Dockerfile#L31..L34

Then we'd likely get a more tested virtualbmc version than relying on pip install where there is no CI/promotion pipeline?

Any thoughts on that @elfosardo @maelk ?

Reply:
No objections from my side, unless people will be sad about switching to CentOS.

Reply:
the CI/promotion of vbmc comes from the openstack testing infrastructure, so it's not like we don't get any testing
not against using RDO pkgs, will probably get better integration testing with that and the centos8 repos are very close to be finally stable, so we can definitely consider that